.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -30,6 +30,39 @@ def integration_suite_extra_cmds_fn(version):
     ]
 
 
+def backcompat_suite_extra_cmds_fn(release_mapping):
+    dagit_version = release_mapping["dagit"]
+    user_code_version = release_mapping["user_code"]
+
+    if dagit_version == "current_branch":
+        dagit_dockerfile = "./Dockerfile_dagit_source"
+    else:
+        dagit_dockerfile = "./Dockerfile_dagit_release"
+
+    if user_code_version == "current_branch":
+        user_code_dockerfile = "./Dockerfile_grpc_source"
+    else:
+        user_code_dockerfile = "./Dockerfile_grpc_release"
+
+    def _extra_cmds_fn(_):
+        return [
+            f'export DAGIT_DOCKERFILE="{dagit_dockerfile}"',
+            f'export USER_CODE_DOCKERFILE="{user_code_dockerfile}"',
+            "pushd integration_tests/test_suites/backcompat-test-suite/dagit_service",
+            f"./build.sh {dagit_version} {user_code_version}",
```

Comment:
it would be great if there was a straightforward way to run these locally too (could just be a README)